### PR TITLE
Add employee front end

### LINKED ISSUE #56: new employee
## New employee tap 

- [ ] Route ` /dashboard`

---

- [ ] Description 
### You can see:

- img

- 3 text fields for name ,email and date of hiring
- selector to select position
- button to confirm adding process

- [ ] UI
Insert the new employee

### PR DESCRIPTION
**Pull request by rubasider**
### What does this PR do?
Call the add new employee with http://localhost:3000/dashboard endpoint.

* On the dashBoard page you can add new employee  to the
  database after you add his name,email,date of hiring date and position.

*  button to confirm adding the employee.

- a pop up will appear to tell you if the process is done successfuly usin
  sweetalert.

### Does this Fix an Issue?

Fixes #56

### How to test this PR

- npm i

- cd client && npm i

- cd..

- npm run dev
- go to http://localhost:3000/dashboard endpoint

### Screenshots of changes
https://ibb.co/1fJgHrv

### Screenshots of changes

### CODE REVIEW

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 1]
```diff
@@ -110,56 +123,147 @@ export default function AddNewMember() {
...
+									>
+										اضافة
+									</Button>
+								</Box>
+							</Box>
+						</div>
+					</form>
+				</Grid>
+			</Grid>
+		</Box>
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx]: I can`t read the code in this way! use formatting such as linting or prettier!!

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 2]
```diff
@@ -110,56 +123,147 @@ export default function AddNewMember() {
...
+											required={true}
+											error={!empData.position.isValid}
+											autoFocus={true}
+											color="secondary"
+											onChange={handleChange}
+											inputProps={{
+												name: 'position',
+											}}
+										>
+											<option aria-label="None" value="" />
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx]: Wondering why!

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 3]
```diff
@@ -110,56 +123,147 @@ export default function AddNewMember() {
...
+								<Box m={2}>
+									<TextField
+										className={classes.fullWidth}
+										required={true}
+										error={!empData.email.isValid}
+										autoFocus={true}
+										name="email"
+										id="standard-required"
+										placeholder="الايميل"
+										onChange={handleChange}
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx:170]: what if the user enters a name in email input, I mean not email formatting, should that be okay?

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/components/LayoutComponents/Dashboard/NewMember.jsx [hunk 4]
```diff
@@ -55,6 +55,10 @@ export default function AddNewMember() {
 		},
 	});
 	const classes = NewMemberStyle();
+	const validateEmail = (email) => {
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Dashboard/NewMember.jsx:58]: regular expressions! can you document this?

**Review by yaqootturman — DISMISSED** [review]
Great work but I need you to document regular expressions!

**Review by karmelyoei — DISMISSED** [review]

**Review by karmelyoei — APPROVED** [review]

**Comment by mantagen** [comment]
@rubasider you can paste image directly into PR description (you don't have use external website)

**Review by mantagen — COMMENTED** [review]
The styles look really good!!

Only one problem i have a problem when submitting the form:
image
image

**Comment by rubasider** [comment]
it works fine on my side

**Pull request closed without merging**